### PR TITLE
Update our JS to support IE11 and old browsers

### PR DESCRIPTION
**Pull request by mbenadda**
## Purpose

Richie is not IE11 ready. One of the major parts that did not work with outdated browsers was the JS, which needed to be built specifically to support it.

## Proposal

- [x] add a browserslist with our target browsers
- [x] build dependencies that can't run in IE11
- [x] add required polyfills for IE11
- [x] update `base.html` JS so it runs in IE11

### CODE REVIEW

File: src/richie/apps/core/templates/richie/base.html [hunk 1]
```diff
@@ -110,26 +110,29 @@
...
-                $topbarBurgers.forEach( el => {
-                    el.addEventListener('click', () => {
-                        // Get the target from the "data-target" attribute
-                        const target = el.dataset.target;
-                        const $target = document.getElementById(target);
+                $topbarBurgers.forEach(function(el) {
+                  el.addEventListener("click", function() {
+                    // Get the target from the "data-target" attribute
+                    const target = el.dataset.target;
+                    const $target = document.getElementById(target);
```
Comment (lunika) [inline src/richie/apps/core/templates/richie/base.html]:
```suggestion
                    const target = document.getElementById(el.dataset.target);
```

File: src/richie/apps/core/templates/richie/base.html [hunk 2]
```diff
@@ -110,26 +110,29 @@
...
                 // Add a click event on each of them
-                $topbarBurgers.forEach( el => {
-                    el.addEventListener('click', () => {
-                        // Get the target from the "data-target" attribute
-                        const target = el.dataset.target;
-                        const $target = document.getElementById(target);
+                $topbarBurgers.forEach(function(el) {
+                  el.addEventListener("click", function() {
+                    // Get the target from the "data-target" attribute
+                    const target = el.dataset.target;
```
Comment (lunika) [inline src/richie/apps/core/templates/richie/base.html]: you can remove this line. this comment for next line.

File: src/richie/apps/core/templates/richie/base.html (hunk 1 above)
Comment (lunika) [reply src/richie/apps/core/templates/richie/base.html]: Also rename `$target` in `target`. We are not doint php ;-)

File: src/richie/apps/core/templates/richie/base.html [hunk 3]
```diff
@@ -110,26 +110,29 @@
         {% render_block "js" %}
         <script src="{% static 'richie/js/index.js' %}"></script>
         <script>
-        document.addEventListener('DOMContentLoaded', () => {
-            // Get all topbar burger elements
-            const $topbarBurgers = Array.prototype.slice.call(document.querySelectorAll('.topbar__hamburger'), 0);
+            document.addEventListener("DOMContentLoaded", function() {
+              // Get all topbar burger elements
+              const $topbarBurgers = Array.prototype.slice.call(
```
Comment (lunika) [inline src/richie/apps/core/templates/richie/base.html]: why having a `$` at the beginning of your variable name ?

**Comment by mbenadda** [comment]
I fixed the base.html JS issues you pointed out 👍

**Pull request merged**